src/Api/ItemDataProviderInterface.php

```diff
@@ -22,8 +23,6 @@
 interface ItemDataProviderInterface
 {
     /**
-     * Retrieves an item.
```

Comment:
The interface description is `"Retrieves items from a persistence layer."`, so having `"Retrieves an item."` for this interface `getItem()` method description looks... obvious.

That being said it's just a method description... No strong feelings here :)